mlir/include/mlir/Dialect/Vector/Transforms/VectorRewritePatterns.h

```diff
@@ -148,6 +148,10 @@ void populateVectorTransferCollapseInnerMostContiguousDimsPatterns(
 void populateSinkVectorBroadcastPatterns(RewritePatternSet &patterns,
                                          PatternBenefit benefit = 1);
 
+/// Patterns that re-order transpose Ops.
+void populateReoderVectorTransposePatterns(RewritePatternSet &patterns,
+                                           PatternBenefit benefit = 1);
```

Comment:
It would be good to explain in the the docs _why_ you may want to use these patterns. This line doc comment does not give me any more information than the function name.